DigitalLearningSolutions.Web/Views/SuperAdmin/Centres/_CentreCard.cshtml

```diff
@@ -0,0 +1,90 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Centres
+@model CentreSummaryViewModel
+
+<div class="searchable-element nhsuk-expander-group" id="@Model.CentreId-card">
+  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.CentreId-name" name="centre-name">
+        @Model.CentreName (@Model.CentreId)
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+      <dl class="nhsuk-summary-list">
+
+        <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Centre id

```

Comment:
ID is capitalised on the delegates lists, so I think we should keep that consistent. Appreciate lower case matches the wireframes. See https://www.dls.nhs.uk/dev-my-learning-portal/TrackingSystem/Delegates/CourseDelegates

Reply:
Done.

Reply:
I think D should be capital as well.

Reply:
Done.